### DISCUSSION TITLE
GET / MySQL Node.js

### DISCUSSION THREAD

**Discussion by acrossmindanduniverse** [flant/grafana-statusmap#188] [Q&A]
I'm a newbie to mysql, so is there possible to GET data by row and GET data from all of my tables in one function?
here's my code:

 getAllBooksModel: function(detail) {
    return new Promise ((resolve, reject) => {
      if (detail) {
        connection.query('SELECT * FROM books WHERE ?', [detail], function (err, res) {
          if (err) {
            reject(err)
          } else {
          resolve(res)
          console.log(res)
        }
      })
      } else {
        connection.query('SELECT * FROM books', function (err, res) {
          if (err) {
            reject(err)
          } else {
          resolve(res)
          console.log(res)
        }
      })
      }
    })
    }

**Comment by diafour** [comment]
Sorry, this is not a nodejs supporting forum. Please ask on Stackoverflow.
